.eleventy.js

```diff
@@ -46,8 +46,6 @@ module.exports = function (eleventyConfig) {
 
   eleventyConfig.setLibrary('md', markdown);
 
-  eleventyConfig.setUseGitIgnore(false);
```

Comment:
unrelated?

Reply:
`fix: do not reload eleventy infinitely` – else when I tried this out it infinitely retried in the background until Open Collective ratelimited me